es6/test/lib/createRouteHandler.js

```diff
@@ -0,0 +1,339 @@
+'use strict';
+//jshint camelcase: false
+//jshint maxstatements: false
+
+var assert = require('assert');
+var Bluebird = require('bluebird');
+var sinon = require('sinon');
+
+describe('createRouteHandler', function() {
+  var routeHandler;
```

Comment:
If they're mutated, use `let` not `var`. `options` can probably just be defined as a `const`